raiden/raiden_event_handler.py

```diff
@@ -455,14 +463,21 @@ def handle_contract_send_channelupdate(
             )
             our_signature = raiden.signer.sign(data=non_closing_data)
 
-            channel.update_transfer(
-                nonce=balance_proof.nonce,
-                balance_hash=balance_proof.balance_hash,
-                additional_hash=balance_proof.message_hash,
-                partner_signature=balance_proof.signature,
-                signature=our_signature,
-                block_identifier=channel_update_event.triggered_by_block_hash,
-            )
+            try:
+                channel.update_transfer(
+                    nonce=balance_proof.nonce,
+                    balance_hash=balance_proof.balance_hash,
+                    additional_hash=balance_proof.message_hash,
+                    partner_signature=balance_proof.signature,
+                    signature=our_signature,
+                    block_identifier=channel_update_event.triggered_by_block_hash,
+                )
+            except InsufficientEth as e:
+                raise RaidenUnrecoverableError(
+                    f"{str(e)}\n"
+                    "CAUTION: This happened during a channel settlement. You "
```

Comment:
If you want such an error here be more specific. "When updating our side of the channel within the channel settlement period"